charts/tfy-karpenter-config/values.yaml

```diff
@@ -37,6 +37,8 @@ karpenter:
     - alias: al2023@latest
     ## @param karpenter.defaultNodeTemplate.userData User data to append in the default node template. Set this to "" to disable injection.
     userData: ""
```

Comment:
Should we unify this with `gpuDefaultNodeTemplate`  to make `userData` default as `"default"` ?